sunpy/data/data_manager/cache.py

```diff
@@ -167,5 +167,6 @@ def download(url):
                 return download(url)
             except Exception as e:
                 warn(e, SunpyUserWarning)
```

Comment:
Is there a reason we are using `warnings.warn` and not the logger here?

Reply:
I don't remember a particular reason. I think I might have copied the pattern from other (old?) parts of the code.